config/envs/default.py

```diff
@@ -5,6 +5,7 @@
 from config.mappings import assessment_mapping
 from fsd_utils import CommonConfig
 from fsd_utils import configclass
+from fsd_utils import NotifyConstants
```

Comment:
This came from a merge from main. Not sure why its showing here

Reply:
Doesn't look like it's used anywhere so you can probably delete it